src/domains/job/search/detail/components/RelatedJobSearchPosts.jsx

```
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { getJobPosts } from '../../../../job/main/service/jobMainService'
+import JobPostList from '../../../main/components/JobPostList'
+import Spinner from '../../../../../components/web/Spinner'
+
+const RelatedJobSearchPosts = ({ currentId }) => {
```

ESLint

[error] 'currentId' is missing in props validation

(react/prop-types)